### DISCUSSION TITLE
Bad midi file , expected 'Mhdr'

### DISCUSSION THREAD

**Discussion by xaviBazalar** [cifkao/html-midi-player#41] [Q&A] [answered]
Hello! I don't know what happened that. The same midi works in the https://cifkao.github.io/html-midi-player/ but in my own website show that message "Bad midi file , expected 'Mhdr'". Help me please!

**Comment by cifkao** [comment, marked as answer]
Hi! Can you provide more details please? What is your setup?

If you are loading your MIDI file from a URL, my guess would be that your server is not serving the file properly, perhaps returning an error page instead. You can use your browser's developer tools (Network tab) to check what file is actually being transferred.

**Reply by timcharlier6** [reply]
Hi! 
I have the same problem. 
Dev tools says :
  - "GET myMid.mid [HTTP/3 404 Not Found 17ms]"
  - "content-type | text/plain; charset=utf-8"

**Reply by cifkao** [reply]
@timcharlier6 Your server is not actually serving the MIDI file, maybe your path is incorrect.

**Reply by cifkao** [reply]
@timcharlier6 I just added a section to the FAQ about this: https://github.com/cifkao/html-midi-player/tree/master#why-is-my-midi-file-not-loading
Please check it and let me know if that solves your problem.

**Reply by timcharlier6** [reply]
Hi. It works with a url it's great, but it doesn't work when I use a local midi file so I will host midi somewhere. Thank you.

**Comment by Coozje** [comment]
I had the same issue. If you load your file from somewhere in the wwwroot folder - like me - make sure to leave out the "~/" at the start of the path in the src attribute.
